docs/client/configure-kn.md

```diff
@@ -9,15 +9,36 @@ You can customize your `kn` CLI setup by creating a `config.yaml` configuration
 ## Example configuration file
 
 ```yaml
+# Plugins related configuration
 plugins:
+  # Whether to lookup configuration in the execution path (default: true). This option is deprecated and will be removed in a future version where path lookup will be enabled unconditionally
   path-lookup: true
+  # Directory from where plugins with the prefix "kn-" are looked up. (default: "$base_dir/plugins"
+  # where "$base_dir" is the directory where this configuration file is stored)
   directory: ~/.config/kn/plugins
+# Eventing related configuration
 eventing:
+  # List of sink mappings that allow custom prefixes wherever a sink
+  # specification is used (like for the --sink option of a broker)
   sink-mappings:
+    # Prefix as used in the command (e.g. "--sink svc:myservice")
   - prefix: svc
+    # Api group of the mapped resource
     group: core
+    # Api version of the mapped resource
     version: v1
+    # Resource name (lowercased plural form of the 'kind')
     resource: services
+  # Channel mappings that you can use in --channel options
+  channel-type-mappings:
+    # Alias that can be used as a type for a channel option (e.g. "kn create channel mychannel --type Kafka")
+  - alias: Kafka
+    # Api group of the mapped resource
+    group: messaging.knative.dev
+    # Api version of the mapped resource
+    version: v1alpha1
```

Comment:
This can be probably update to later version. @pierDipi can you pls confirm?
```suggestion
    version: v1beta2
```

Reply:
KafkaChannel version is at v1beta1

Reply:
and that's the only version https://github.com/knative-sandbox/eventing-kafka-broker/blob/32dfed583ceb02c4fa1581b3bffab07d6ec0e7d6/control-plane/config/eventing-kafka-broker/100-channel/100-kafka-channel.yaml#L27C13-L27C20